pep_sphinx_extensions/pep_zero_generator/writer.py

```diff
@@ -92,6 +99,7 @@ def emit_column_headers(self) -> None:
         self.emit_text("     - PEP")
         self.emit_text("     - Title")
         self.emit_text("     - Authors")
+        self.emit_text("     - ")
```

Comment:
I think a note might be useful here just to avoid future confusion
```suggestion
        self.emit_text("     - ")  # for Python-Version
```